Sources/XcodeProj/Objects/Project/PBXProj.swift

```diff
@@ -66,6 +66,40 @@ public final class PBXProj: Decodable {
             objects: pbxproj.objects
         )
     }
+    
+    /// Initializes the project with the data representation of pbxproj file.
+    ///
+    /// - Parameters:
+    ///   - data: data representation of pbxproj file.
+    public convenience init(data: Data) throws {
```

Comment:
Can we add a simple test to exercise this?

Reply:
Yes sure, I will add a unit test and write back here.

Reply:
@maxwellE Thanks for the suggestion I have added a simple test it just tests if PBXProj is properly initialized with the data representation of the pbxproj file.

```
func test_initialize_PBXProj_with_data() {
    let pbxprojPath = iosProjectPath.url.appendingPathComponent("project.pbxproj")
    
    if let pbxprojStringRepresentation = try? String(contentsOf: pbxprojPath) {
        let pbxprojData = Data(pbxprojStringRepresentation.utf8)
        
        XCTAssertNotNil(try? PBXProj(data: pbxprojData))
        return
    }

    XCTFail()
}
```

Reply:
Thanks!